src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -676,6 +676,21 @@ public static async ValueTask<FormulaValue> SortTable(EvalVisitor runner, EvalVi
             }
         }
 
+        public static FormulaValue TableConcatenate(IRContext irContext, FormulaValue[] args)
+        {
+            var tables = Array.ConvertAll(
+                args,
+                arg => arg switch
+                {
+                    TableValue r => r.Rows,
+                    BlankValue b => new List<DValue<RecordValue>> { DValue<RecordValue>.Of(b) },
+                    ErrorValue e => new List<DValue<RecordValue>> { DValue<RecordValue>.Of(e) },

```

Comment:
This doesn't look right - see comments in the .txt test file

Reply:
Makes sense will fix.